cves/CVE-2011-1183.yml

```diff
@@ -106,8 +104,8 @@ unit_tested:
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
   answer: |
-    The function where the fix exists was not unit tested until the new fix was 
-    implemented. A test was then added after the new fix.
+    While tests did exist for this particular portion of the code, none were specifically looking for a problem
+    like this. After the CVE was fixed a new test was written specifically for this scenario.
   code: false
```

Comment:
I believe that since the original code _was_ unit tested, and this edge case was not accounted for, the answer would be `true`.